### ISSUE TITLE
Google Assistant Compatibility

### ISSUE THREAD

**Issue by ricardonlo** [PaulWoitaschek/Voice#1036]
I'm new here, do you guys think it is something worth/able to do ?

**Comment by finBitorg** [comment]
Yes please add ability to cast audio to Chrome / Assistant devices. Many podcast players have this feature.
